src/command/info/info.c

```diff
@@ -868,10 +876,19 @@ formatTextBackup(const DbGroup *dbGroup, String *resultStr)
         KeyValue *repoInfo = varKv(kvGet(info, INFO_KEY_REPOSITORY_VAR));
 
         strCatFmt(
-            resultStr, "            repo%u: backup set size: %s, backup size: %s\n",
+            resultStr, "            repo%u: backup set size: %s",
             varUInt(kvGet(varKv(kvGet(backupInfo, KEY_DATABASE_VAR)), KEY_REPO_KEY_VAR)),
-            strZ(strSizeFormat(varUInt64Force(kvGet(repoInfo, KEY_SIZE_VAR)))),
-            strZ(strSizeFormat(varUInt64Force(kvGet(repoInfo, KEY_DELTA_VAR)))));
+            strZ(strSizeFormat(varUInt64Force(kvGet(repoInfo, KEY_SIZE_VAR)))));
+
+        if (kvGet(repoInfo, KEY_SIZE_MAP_VAR) != NULL)
+            strCatFmt(resultStr, " (%s map)", strZ(strSizeFormat(varUInt64Force(kvGet(repoInfo, KEY_SIZE_MAP_VAR)))));
+
+        strCatFmt(resultStr, ", backup size: %s", strZ(strSizeFormat(varUInt64Force(kvGet(repoInfo, KEY_DELTA_VAR)))));
+
+        if (kvGet(repoInfo, KEY_DELTA_MAP_VAR) != NULL)
+            strCatFmt(resultStr, " (%s map)", strZ(strSizeFormat(varUInt64Force(kvGet(repoInfo, KEY_DELTA_MAP_VAR)))));
```

Comment:
If I understood correctly, key_size_map = size of the map for the entire backup set but the map could reference a file from a previous backup so key_delta_map = only the size of the map for the files in this specific backup set.

Since that info will be displayed to users, wouldn't it be worth it to document it somewhere (in the user-guide perhaps)?

Reply:
Yes, we'll need to do that, but I can't add the docs yet since this will be release initially as an internal feature.

Reply:
Or maybe this is too much even for the detailed output? I added it mostly so we could analyze block size strategies and perhaps the JSON output is good enough for that.

Reply:
Even when only displayed in the backup set text details, I believe we should document everything users can "see" or we'll probably have questions about it.

Probably step 1 would be to add it into JSON output and if we later see it can be used more often, then adding it to the backup set text with the documentation could be step 2?

Reply:
Agreed, reverted this in eed6c4886.

Reply:
Looks good to me :-)